logbook-core/src/main/java/org/zalando/logbook/ChunkingHttpLogWriter.java

```diff
@@ -42,8 +42,13 @@ public void writeRequest(final Precorrelation<String> precorrelation) throws IOE
     @Override
     public void writeResponse(final Correlation<String, String> correlation) throws IOException {
         split(correlation.getResponse()).forEach(throwing(part ->
-                writer.writeResponse(new SimpleCorrelation<>(correlation.getId(), correlation.getDuration(),
-                        correlation.getRequest(), part))));
+                writer.writeResponse(new SimpleCorrelation<>(
```

Comment:
Why not to introduce SimpleCorrelation constructor which takes correlation as an argument?